pkg/jx/cmd/status.go

```diff
@@ -49,6 +49,7 @@ func NewCmdStatus(commonOpts *CommonOptions) *cobra.Command {
 	}
 
 	cmd.Flags().StringVarP(&options.node, "node", "n", "", "the named node to get ")
+	options.addCommonFlags(cmd)
```

Comment:
This is not required anymore. The common flags are added in the root command.

Reply:
fixed